index.html

```diff
@@ -35,5 +35,144 @@ <h1 class="bigtext1">I'm Alexandra <br> Glad to see you!</h1>
+        <p class="card-title">Multi-Post <br> Stories</p>
+        <p class="canopy">
+          <span class="span1">CANOPY</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Backend</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Dev 2015</span></p>
+          <p class="card-text">A daily selection of privately personalised reads: no accounts or sign-ups required.</p>
+          <div class="btns-3">
+            <ul>
+              <li><button type="button" class="btn3">html</button></li>
+              <li><button type="button" class="btn3">css</button></li>
+              <li><button type="button" class="btn3">javascript</button></li>
+            </ul>
+          </div>
+          <div>
+            <button type="button" class="btn-1">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="works">
+      <div class="card-content">
+        <img src="./images/works3_img.png" alt="First works image" class="img-class">
+        <div class="text-border">
+        <p class="card-title">Tonic</p>
+        <p class="canopy">
+          <span class="span1">CANOPY</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Backend</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Dev 2015</span></p>
+          <p class="card-text">A daily selection of privately personalised reads: no accounts or sign-ups required.</p>
+          <div class="btns-3">
+            <ul>
+              <li><button type="button" class="btn3">html</button></li>
+              <li><button type="button" class="btn3">css</button></li>
+              <li><button type="button" class="btn3">javascript</button></li>
+            </ul>
+          </div>
+          <div>
+            <button type="button" class="btn-1">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="works">
+      <div class="card-content">
+        <img src="./images/works4_img.png" alt="First works image" class="img-class">
+        <div class="text-border">
+        <p class="card-title">Multi-Post <br> Stories</p>
+        <p class="canopy">
+          <span class="span1">CANOPY</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Backend</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Dev 2015</span></p>
+          <p class="card-text">A daily selection of privately personalised reads: no accounts or sign-ups required.</p>
+          <div class="btns-3">
+            <ul>
+              <li><button type="button" class="btn3">html</button></li>
+              <li><button type="button" class="btn3">css</button></li>
```

Comment:
- [ ] Kindly make sure you update your project cards design by fixing the overlapping issue of all your cards to stick as much as possible to Figma design, use the same spacing between cards,...
 Below you can see how your cards look:

| Your Design | Your Design |
| ---------------- | ------------------ |
| ![Screenshot from 2023-02-17 06-13-23](https://user-images.githubusercontent.com/83042742/219548206-ab65f1aa-5a71-483b-a504-3a09d9edffd2.png) | ![Screenshot from 2023-02-17 06-13-45](https://user-images.githubusercontent.com/83042742/219548244-b78b667a-6746-4d31-83ce-b54e23d8c494.png) |
| ---------------------- | ------------------- | 
| Figma Design | Figma Design |
| ------------------ | -------------------- |
| ![Screenshot from 2023-02-17 06-20-08](https://user-images.githubusercontent.com/83042742/219548888-39f53583-e850-44db-8e04-f25c1da52bc0.png) |

Reply:
![Cards_not_overlapping_here](https://user-images.githubusercontent.com/104390780/219600663-dcc85660-4934-4e5e-8166-6512f99ba163.png)

May i kindly ask you to help me point out the problem with my code if i have the output I attached on my system and you have the output you sent on your system.

Is there any adjustment i can make to ensure that it shows the same way on your side??